### PR TITLE
[ru] sensor apis

### PR DESCRIPTION
**Pull request by DmitrySkripunov**
Hello! 
Suggesting a translation to Russion Sensor APIs page

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
+yarn.lock
+node_modules
```
Comment (caugner) [inline .gitignore:2]: `translated-content` doesn't have a `package.json`, so these two lines should not be added.

```suggestion
```

**Review by caugner — DISMISSED** [review]

**Comment by sashasushko** [comment]
@DmitrySkripunov thank you for your help. I remember about your PR. And I'll review it ASAP. Sorry for waiting. Peace 🕊

File: .gitignore (hunk 1 above)
Comment (caugner) [reply .gitignore:2]: @DmitrySkripunov In the mean-time, translated-content did get a `package.json`. Would you mind opening a separate PR for adding these lines to `.gitignore`?

Comment (caugner) [reply .gitignore:2]: Never mind, there already is one: https://github.com/mdn/translated-content/pull/9269

**Comment by queengooborg** [comment]
Hey @sashasushko @myshov, just checking in to see if you plan to review this PR soon!

**Comment by queengooborg** [comment]
Since this PR has been sitting around for over a month, I have updated the PR and am going to merge it.  I apologize if there are inaccuracies in the translation (I don't speak Russian), but it does not feel right to simply close the PR and waste the contributor's efforts.

**Comment by sashasushko** [comment]
There are small issues with punctuation, but in total the translation is ok. Thanks, @queengooborg

**Pull request merged**